ansible_builder/main.py

```diff
@@ -46,6 +52,7 @@ def build_command(self):
         ]
 
     def build(self):
+        self.containerfile.initialize_folder()
```

Comment:
Key point here:

Adding a call at the start of `.build` that will create the build context folder, and also copy files into it.

That's the big organizational change that most of the rest of this diff supports.

Reply:
Not crazy about this method name. `initialize_build_context`?

Reply:
it creates the folder and copies files, so what about `create_folder_copy_files`

I'm fine with any of the names mentioned so far.